recorder-sample.yaml

```diff
@@ -0,0 +1,43 @@
+replicaCount: 1
+
+recorder:
+  log_level: info
+  health_port: 8080
+  redis:
+    address: <redis_host:port>
+    # db: 0
+    # username:
+    # password:
+  # api key+secret required if recording with templates by room name only
+  apiKey: "server-api-key"
+  apiSecret: "server-api-secret"
+  # default input options
+  input:
+    width: 1920
+    height: 1080
+    depth: 24
+    framerate: 25
```

Comment:
why do we default to fps of 25 instead of 30?

Reply:
24/25 is the minimum for smooth looking video

Reply:
we can try recording with both and see if there's a noticeable difference

Reply:
I think 30fps is commonly used due to US video standard. I've rarely seen 24 for online video
![0E8FFBB8-FBAA-4AB2-A29C-822CB5A83914](https://user-images.githubusercontent.com/320582/127826820-bca124f1-5f22-4411-9447-d88f3404df10.jpeg)